doc/rst/usingchapel/chplenv.rst

```diff
@@ -718,15 +718,18 @@ CHPL_LIB_PIC
 
 Character Set
 -------------
-   Chapel is tested with the Unicode character set and the traditional
-   C collating sequence using the following settings.
+   We have the most experience running Chapel with the Unicode
+   character set and the traditional C collating sequence using the
+   following settings.
 
    .. code-block:: sh
 
        LANG=en_US.UTF-8
        LC_COLLATE=C
        LC_ALL=""
 
+   .. note::
+       This may change in the future.
```

Comment:
Might want to be more specific to say something like "Other settings might be recommended in the future". The word *This* is just too nonspecific for me.

Reply:
Good point.